src/Entity/Page.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\UuidInterface;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity()
+ */
+class Page
+{
+    /**
+     * @ORM\Id()
+     * @ORM\Column(type="guid")
+     */
+    private $id;
+
+    /**
+     * @Assert\NotBlank()
```

Comment:
Do we need to add @Assert in the entity ? Values are already check in the DTO 
What do you think @tdutrion ?

Reply:
In here nullable=false could be added to ORM\Column, it will add constraints on the Page table